src/shared/sam/debugger/goSamDebug.ts

```diff
@@ -179,7 +179,7 @@ async function installDebugger(debuggerPath: string): Promise<void> {
     const isWindows: boolean = os.platform() === 'win32'
     const installScriptPath: string = await makeInstallScript(debuggerPath, isWindows, false)
 
-    const childProcess = new ChildProcess(installScriptPath)
+    const childProcess = new ChildProcess(false, installScriptPath)
```

Comment:
In general I'd suggest logging most commands, if they are not very noisy. This only runs under certain conditions so it's probably worth logging?